integration/basic_test.go

```diff
@@ -248,9 +248,17 @@ func TestDatabaseName(t *testing.T) {
 	t.Run("Empty", func(t *testing.T) {
 		ctx, collection := setup.Setup(t)
 
-		// there is no explicit command to create database, so create collection instead
 		err := collection.Database().Client().Database("").CreateCollection(ctx, testutil.CollectionName(t))
 		expectedErr := driver.InvalidOperationError(driver.InvalidOperationError{MissingField: "Database"})
 		assert.Equal(t, expectedErr, err)
 	})
+
+	t.Run("63ok", func(t *testing.T) {
+		ctx, collection := setup.Setup(t)
+
+		dbName63 := strings.Repeat("a", 63)
+		err := collection.Database().Client().Database(dbName63).CreateCollection(ctx, testutil.CollectionName(t))
+		require.NoError(t, err)
+		collection.Database().Client().Database(dbName63).Drop(ctx)
```

Comment:
That happens automatically on test teardown, no need to do that there

Reply:
> That happens automatically on test teardown, no need to do that there

Not in this case: it's a specific database that was created in the test.
If I don't delete it, `TestEmbedded` fails, because it's presence.